packages/ui-library/src/components/dashboard/index.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useMemo, useState } from 'react';
+import { Cell, Label, Pie, PieChart, ResponsiveContainer } from 'recharts';
+import styled from 'styled-components';
+import orderBy from 'lodash/orderBy';
+import { BackgroundPaper, ContainerBox, Grid, Popper, Typography } from '..';
+import { LinearProgress, linearProgressClasses, useTheme } from '@mui/material';
+import { colors } from '../../theme';
+import DashboardPopper from './popper';
+
+const StyledBackgroundPaper = styled(BackgroundPaper)`
+  ${({ theme: { spacing } }) => `
+    padding: ${spacing(5)};
+  `}
+  display: flex;
+  flex: 1;
+`;
+
+const StyledBullet = styled.div<{ fill: string }>`
+  ${({ fill, theme: { spacing } }) => `
+    width: ${spacing(2)};
+    height: ${spacing(2)};
+    border-radius: ${spacing(10)};
+    ${fill && `background-color: ${fill};`}}
+  `}
+`;
+
+const BorderLinearProgress = styled(LinearProgress)<{ fill: string }>(
+  ({
+    theme: {
+      palette: { mode },
+      spacing,
+    },
+    fill,
+  }) => ({
+    background: colors[mode].background.primary,
+    height: spacing(2),
+    [`& .${linearProgressClasses.bar}`]: {
+      borderRadius: spacing(3),
+      background: fill,
+    },
+  })
+);
+
+interface BaseData {
+  name: string;
+  value: number;
+}
+
+interface OtherData extends BaseData {
+  breakdown: Data[];
+  isOther: true;
+}
+
+type Data = BaseData | OtherData;
+
+interface DashboardProps {
+  data: Data[];
+  withPie?: boolean;
+  valuesForOther?: number;
+  valueFormatter?: (value: number) => string;
+}
+
+type DataWithFill = Data & {
+  fill: string;
+  relativeValue: number;
+};
+
+const COLOR_PRIORITIES = {
```

Comment:
Found something interesting... If you change the theme mode in the application, colors in the charts doesn't change. Only in first render they are defined